test/e2e/framework/framework.go

```diff
@@ -382,7 +382,8 @@ func (f *Framework) AfterEach() {
 		if TestContext.DeleteNamespace && (TestContext.DeleteNamespaceOnFailure || !ginkgo.CurrentGinkgoTestDescription().Failed) {
 			for _, ns := range f.namespacesToDelete {
 				ginkgo.By(fmt.Sprintf("Destroying namespace %q for this suite.", ns.Name))
-				if err := f.ClientSet.CoreV1().Namespaces().Delete(context.TODO(), ns.Name, metav1.DeleteOptions{}); err != nil {
+				err := f.ClientSet.CoreV1().Namespaces().Delete(context.TODO(), ns.Name, metav1.DeleteOptions{})
```

Comment:
Nit: why this change? The previous line did the same thing.

Reply:
this code actually got reverted. You might have commented on outdated diff. Let me squash the commits, so as it goes away entirely.

Reply:
So I have squashed the commits but I still wanted changes into `AfterSuite` hook ordering in separate commit. This does mean some of old bits are still in that commit which gets dropped in second commit.

Reply:
Alright. Fixed up first commit to drop changes into framework files.